Add guidance on not using centred text (#5495)
alphagov/govuk-design-system: src/styles/font-override-classes/index.md

- `govuk-!-text-align-right` to align text to the right
- `govuk-!-text-align-centre` to align text to the centre

> As the most commonly used languages on GOV.UK are written left-to-right, you should usually left-align body copy text. Right-aligned body copy text can be hard for users to find and read when they magnify their screen. Centre aligned body copy can also be difficult for users who magnify their screen, with multi-line text requiring the user to find the start of each line in a different place. Use right-aligned body copy text if you are translating content into a language which is written right-to-left.

Reviewer: It's a bit of a chunky paragraph to do at once. Also think we can combine the issue with centre and right-aligned text into the same paragraph.

How's this to make things a bit more readable? 

```suggestion
As the most commonly used languages on GOV.UK are written left-to-right, you should usually left-align body copy text.

Centre and right-aligned body copy text can be hard for users to find and read when they magnify their screen, as these require the user to find the start of each line in different place.

Use right-aligned body copy text if you are translating content into a language which is written right-to-left.
```
